src/code.rs

```diff
@@ -20,6 +20,7 @@ use std::error::Error;
 #[non_exhaustive]
 pub enum Code {
     /// <kbd>`~</kbd> on a US keyboard. This is the <kbd>半角/全角/漢字</kbd> (<span class="unicode">hankaku/zenkaku/kanji</span>) key on Japanese keyboards
+    /// This is also called a backtick or grave.
     Backquote,
```

Comment:
Would it be worth using `#[doc(alias = "..")]` here? (ehttps://std-dev-guide.rust-lang.org/policy/doc-alias.html)

Reply:
Opened https://github.com/pyfisch/keyboard-types/issues/64 for that.